kubernetes_practice/resources/server.py

```diff
@@ -0,0 +1,41 @@
+from http.server import BaseHTTPRequestHandler, HTTPServer
+
+"""
+This is a sample http server
+"""
+
+class Request_handler(BaseHTTPRequestHandler):
+    def _set_headers(self):
+        self.send_response(200)
+        self.send_header('Content-type', 'text/html')
+        self.end_headers()
+
+    def do_GET(self):
+        self._set_headers()
+        self.wfile.write(bytes("<p>hello</p>", "utf-8"))
+
+    def do_HEAD(self):
```

Comment:
Similarly, => `do_head`.

As a general rule of thumb, methods in python are always lower case and snake_case.